package/src/contexts/pollContext/createPollContentContext.tsx

```diff
@@ -1,14 +1,12 @@
 import React, { PropsWithChildren, useContext } from 'react';
 
-import { CreatePollData } from 'stream-chat';
-
 import { MessageInputContextValue } from '../messageInputContext/MessageInputContext';
 import { DEFAULT_BASE_CONTEXT_VALUE } from '../utils/defaultBaseContextValue';
 
 import { isTestEnvironment } from '../utils/isTestEnvironment';
 
 export type CreatePollContentContextValue = {
-  createAndSendPoll: (pollData: CreatePollData) => Promise<void>;
+  createAndSendPoll: () => Promise<void>;
```

Comment:
Please make sure this is documented well, that for sending polls you have to utilize the composer now too. Even better if we can do this for the integrators by utilizing this function as well.